common/reviews/api/heft.api.md

```diff
@@ -14,6 +14,7 @@ import { IPackageJson } from '@rushstack/node-core-library';
 import { ITerminalProvider } from '@rushstack/node-core-library';
 import { SyncHook } from 'tapable';
 import { Terminal } from '@rushstack/node-core-library';
+import * as TRushStackCompiler from '@microsoft/rush-stack-compiler-3.7';
```

Comment:
@D4N14L was able to eliminate this dependency for RSC. Can we also eliminate it for Heft? 

It is an expensive download for a **package.json** dependency, and it seems that we only use it for typing info.